.run/Publish VelaptorTesting (Release Linux).run.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Publish VelaptorTesting (Debug Linux)" type="DotNetFolderPublish" factoryName="Publish to folder">
```

Comment:
Change this to say Release instead of Debug

```suggestion
  <configuration default="false" name="Publish VelaptorTesting (Release Linux)" type="DotNetFolderPublish" factoryName="Publish to folder">
```